core/src/main/scala/kafka/server/ForwardingManager.scala

```diff
@@ -43,6 +43,21 @@ object ForwardingManager {
   ): ForwardingManager = {
     new ForwardingManagerImpl(channelManager)
   }
+
+  private[server] def buildEnvelopeRequest(request: RequestChannel.Request,
```

Comment:
Could we use `RequestContext` as the argument instead of `Request`? As far as I can tell, that is all we need. Similarly, we can change `AutoTopicCreationManager.createTopics`.